### PR TITLE
Refactor send batch async

### PR DESCRIPTION
**Pull request by charlyraffellini**
This is a small refactor in order to send messages concurrently.

Here you can see another implementation:
image

### CODE REVIEW

File: Ve.Messaging.Azure.ServiceBus/Publisher/MessagePublisher.cs [hunk 1]
```diff
@@ -34,10 +35,14 @@ public MessagePublisher(IPublisherClientResolver publisherClientResolver)
 
         public async Task SendBatchAsync(IEnumerable<Message> messages)
         {
+            ICollection<Task> tasks = new List<Task>();
+
             foreach (var message in messages)
```
Comment (dave-hillier) [inline Ve.Messaging.Azure.ServiceBus/Publisher/MessagePublisher.cs]: Are there other implementations of this to change? E.g. ThiftPublisher.cs line 27

**Comment by andyroyle** [comment]
can you do me a favour and modify the appveyor.yml file so that it runs the new tests dll that you've created: https://github.com/ve-interactive/Ve.Messaging/blob/master/appveyor.yml#L25

File: Ve.Messaging.Azure.ServiceBus/Publisher/MessagePublisher.cs (hunk 1 above)
Comment (charlyraffellini) [reply Ve.Messaging.Azure.ServiceBus/Publisher/MessagePublisher.cs]: Thanks! 👍

**Comment by charlyraffellini** [comment]
@andyroyle, Done. Here an appveyor screenshot:
image

**Pull request merged**